### PR TITLE
move CODEOWNERS to .github

### PR DESCRIPTION
**Pull request by K-Mistele**

### CODE REVIEW

**Comment by K-Mistele** [comment]
Adding me as the only codeowner, in combination with the branch protection rules, means that I'm required to review all pull requests before they can be merged.

**Review by aleksei-yudin — APPROVED** [review]

**Review by vm-vhryschenko — APPROVED** [review]

**Pull request merged**
